scanner/reader.go

```diff
@@ -0,0 +1,53 @@
+package scanner
+
+import (
+	"errors"
+	"os"
+
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio/detector"
+	"github.com/brimsec/zq/zng/resolver"
+)
+
+type Reader interface {
+	zbuf.Reader
+	Close() error
+}
```

Comment:
Nit: I don't think there's a need for this interface. OpenFiles should just return a *Combiner, and OpenFile should return a *File (currently called fileReader).

Reply:
So rename `fileReader` -> `File`?

Reply:
Yes, that's what I was thinking.

Reply:
Might be clearer. Ok.

Reply:
Yeah, I like it.